config/config.defaults.yml

```diff
@@ -143,16 +143,20 @@ default: &default_settings
   grid_listings_limit: 24
   map_listings_limit: 150
 
+  # DEPRECATED, use Google Tag Manager instead
   # Google Analytics can be used to track traffic on the site
   # If set to true you need to obtain your own API key for analytics
   # and enter it below
   use_google_analytics: false
   google_analytics_key: "enter_your_key_here"
 
-  # You can also use Google Tag Manager
+  # Google Tag Manager can be used to track traffic on the site
+  # If set to true you need to obtain your own container key
+  # and enter it below
```

Comment:
Would it make sense to log deprecation warning if these are still in use? For example, add a piece of code to some file that is loaded on during the app startup (e.g. application.rb or add a new initializer) and add a warning:

```ruby
ActiveSupport::Deprecation.warn("Kissmetrics is deprecated ... ") if APP_CONFIG.use_kissmetrics
```